config/config.go

```diff
@@ -30,6 +30,11 @@ type bot struct {
 }
 
 type db struct {
```

Comment:
Is it really necessary to create two different struct to hold a database path ? What are you trying to achieve here ?

Reply:
db struct is generic DB config, bdb is BadgerDB specific config. For now it might seems useless, but if we add support to more DBs later it'll keep things organized!